Section 1: Introduction and Setup/4-RoR Development Environment/testing_project/Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
```

Comment:
what is this file for?

Reply:
`Rake` is a task runner for Ruby. Rails provides the predefined Rake tasks for creating databases, running migrations, and performing tests. When running `rake`, it will search for `Rakefile` in the parents folder. Tasks can be written inside `Rakefile` or by adding additional rake files in the `lib/tasks` directory with file extension `.rake`.

Reply:
What is a `task runner`?